README.md

```diff
@@ -1,3 +1,97 @@
+- `yarn test:watch`: Run Jest tests in watch mode
+
+Other (you won't use these often):
+
+- `yarn format`: Autoformat all code using Prettier
+- `yarn lint`: Runs the TypeScript compiler and ESLint and outputs issues
+- `yarn test`: Run Jest tests once
+- `yarn storybook:build`: Build Storybook to a static site that can be deployed
+- `yarn build`: Builds the NextJS production asset bundle
+- `yarn start`: Starts the NextJS server in production mode
+- `yarn build:static`: Build the static site asset bundle for deploy
+- `yarn build:analyze`: Builds the asset bundle and generates webpack stats files
+
+### Working on an issue
+
+To begin working on an issue, make sure you've assigned yourself to the issue in Github and marked it as "In Progress.".
+
+Once you have an issue to work on, create a new branch off `main` using the naming convention:
+
+`{issue #}-{short description}`
+
+For example: `112-logo-component`
+
+We have a pre-commit hook set up using Husky to run on staged files only. This will run [Prettier](https://prettier.io/), [TypeScript compilation](https://www.typescriptlang.org/) and [eslint](https://eslint.org/) and fail on errors. For an optimal developer experience, it's recommended that you configure your editor to run linting & formatting inline.
+
+When your branch is ready, open a PR against `main`, fill out the description and request code reviews. The code must pass the same linting and formatting checks mentioned above, as well as [Jest tests](https://jestjs.io/) in order to merge.
+
+### PR Linting
+
+In order to easily maintain a clear changelog, we require all PRs into the `main` branch to pass the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
+
+The format for PR commits is:
+
+```
+<type>(scope): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+#### Allowed `type`s are:
+
+- `build`: Changes that affect the build system or external dependencies (example scopes: webpack, npm)
+- `chore`: Completing a task that has no effective code changes, such as updating the version and changelog for a release
+- `ci`: Changes to our CI configuration files and scripts (example scopes: Circle, Github actions/workflows)
+- `docs`: Documentation only changes
+- `feat`: A new feature
+- `fix`: A bug fix
+- `perf`: A code change that improves performance
+- `refactor`: A code change that neither fixes a bug nor adds a feature
+- `revert`: If the commit reverts a previous commit, it should begin with `revert:` , followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
+- `style`: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `test`: Adding missing tests or correcting existing tests
+
+#### `scope` is optional, and can be one of the following (this list is not exhaustive):
```

Comment:
What's the intent of these. These feel a bit redundant to the above `types`.

Reply:
it's just meta data to provide additional context. For example it may be helpful to know that a bug `fix` applies only to Storybook. it's not necessary but I included it because it's part of https://www.conventionalcommits.org/en/v1.0.0/#specification